localstack/services/internal.py

```diff
@@ -141,6 +141,17 @@ def on_get(self, request):
         return deploy_html
 
 
+class SesResource:
```

Comment:
nit: I think it would be better to add this resource as part of the SES provider itself, to keep a clear separation between the generic internal resources and the service-specific resources. This way we can also benefit from the lazy service loading and will enable (and process) this route only if the SES service has already been activated.

We could make `SesProvider` extend from `ServiceLifecycleHook` and then use the `on_after_init(..)` lifecycle method:  
```
    def on_after_init(self):
        get_internal_apis().add("/ses", SesResource())
```

---
Side note (unrelated to your changes): If we apply the same thinking to `/cloudformation/deploy`, then it should also be moved to the provider. However, it probably makes sense to make that route available as soon as LocalStack starts up (even before the CloudFormation provider itself has been loaded). So, leaving the `CloudFormationUi` resource in here probably makes sense for now. 👍

Reply:
> However, it probably makes sense to make that route available as soon as LocalStack starts up (even before the CloudFormation provider itself has been loaded)

(just out of curiosity) Could it be done in the `on_before_init()` hook in the Cloudformation provider?

Reply:
Moved the resource to SES provider module, and now initialised in the lifecycle hook :heavy_check_mark:

Reply:
> (just out of curiosity) Could it be done in the on_before_init() hook in the Cloudformation provider?

Yep, technically that could be done - the problem is that the `http://localhost:4566/_localstack/cloudformation/deploy` Web UI would then only be accessible after the first CloudFormation API call (e.g., `CreateStack`) has been performed by the user. However, a common use case would be to start up LocalStack, then open the CloudFormation UI, and then trigger a stack deployment from there. (so, it's a bit of a chicken-and-egg problem, whereas in the case of SES we can assume that messages have been sent before we retrieve them via the internal `/_localstack/ses` endpoint...)